.gitignore

```diff
@@ -1,4 +1,5 @@
 .nyc_output
+.nyc_cache
```

Comment:
> ESLint does this by default

It looks to me like ESLint's default behavior is similar to this one.

> Path to the cache location. Can be a file or a directory. If none specified .eslintcache will be used. The file will be created in the directory where the eslint command is executed.

I agree, it is annoying to have to ignore it every time. I am not sure defaulting to the home directory is the best choice though. One advantage of the annoying folder in your project's base directory is that you notice it, and it becomes obvious what it is for. Hiding it away in your home directory puts it out of mind when you are trying to troubleshoot problems. Maybe we could put it in `./node_modules/nyc/.nyc_cache`, that way they end up clearing the cache if they rimraf `node_modules` while troubleshooting.